apiserver/logsink/logsink.go

```diff
@@ -112,6 +133,10 @@ const (
 // ServeHTTP implements the http.Handler interface.
 func (h *logSinkHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	handler := func(socket *websocket.Conn) {
+		h.metrics.TotalConnections().Inc()
```

Comment:
Remember that we desperately want the log counts per model, and ping failures per model.